src/homework21/App.java

```diff
@@ -0,0 +1,85 @@
+package homework21;
+
+import homework21.enums.Type;
+
+import java.time.LocalDate;
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+public class App {

```

Comment:
It seems you haven't implemented tasks 3.2-3.3, so the mark will be lowered for that.